dist/documentation/core-dark-colors.js

```diff
@@ -0,0 +1,267 @@
+/**
+ * Do not edit directly
+ * Generated on Wed, 25 Jan 2023 17:56:51 GMT
+ */
+
+module.exports = {green : {100:{
```

Comment:
looking beyond the formatting weirdness - do we like how these files are generated?

i've made it so it was generate for everything, with these core colours we could decide to have some colour tiles on the design side of the site if we wanted

Reply:
Good call, love this!